packages/babel-traverse/src/scope/index.ts

```diff
@@ -783,7 +783,7 @@ export default class Scope {
   }
 
   buildUndefinedNode() {
```

Comment:
Do you think we could get rid of this in Babel 8?

Reply:
Given that `t.buildUndefinedNode` will be likely introduced in the next minor, if we remove `scope.buildUndefinedNode`, it will break many plugins with very short transition period. I suggest we deprecate `scope.buildUndefinedNode` from the next minor and remove it in Babel 9, if Babel 8 is to be released soon.